content/influxdb3/core/extended-plugin.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Extend plugins with API features and state management
+description: |
+  The Processing engine includes API capabilities that allow your plugins to interact with InfluxDB data and maintain state between executions.
```

Comment:
```suggestion
  The Processing engine includes API capabilities that allow your plugins to interact with your data and maintain state between executions.
```

We always want to be clear that it's the customer's data and we give them the tools to collect, store, query, and process it.